### PR TITLE
Add support for AXIS Gear window shade motor

### PR DESCRIPTION
**Pull request by snjoetw**
This is to add support for AXIS gear (https://www.helloaxis.com).

Here's what the DB entry looks like:

```json
{
  "id": 31,
  "type": "EndDevice",
  "ieeeAddr": "...",
  "nwkAddr": 59277,
  "manufId": 4250,
  "manufName": "AXIS",
  "powerSource": "Mains (single phase)",
  "modelId": "Gear",
  "epList": [1, 196],
  "status": "offline",
  "joinTime": null,
  "endpoints": {
    "1": {
      "profId": 260,
      "epId": 1,
      "devId": 512,
      "inClusterList": [0, 1, 4, 5, 6, 8, 32],
      "outClusterList": [],
      "clusters": {
        "genBasic": {
          "dir": {
            "value": 1
          },
          "attrs": {
            "zclVersion": 1,
            "appVersion": 140,
            "stackVersion": 2,
            "hwVersion": 0,
            "manufacturerName": "AXIS",
            "modelId": "Gear",
            "powerSource": 1
          }
        },
        "genPowerCfg": {
          "dir": {
            "value": 1
          },
          "attrs": {}
        },
        "genGroups": {
          "dir": {
            "value": 1
          },
          "attrs": {
            "nameSupport": 0
          }
        },
        "genScenes": {
          "dir": {
            "value": 1
          },
          "attrs": {}
        },
        "genOnOff": {
          "dir": {
            "value": 1
          },
          "attrs": {}
        },
        "genLevelCtrl": {
          "dir": {
            "value": 1
          },
          "attrs": {
            "currentLevel": 0
          }
        },
        "genPollCtrl": {
          "dir": {
            "value": 1
          },
          "attrs": {
            "checkinInterval": 0,
            "longPollInterval": 20,
            "shortPollInterval": 2,
            "fastPollTimeout": 40,
            "checkinIntervalMin": 180,
            "longPollIntervalMin": 4,
            "fastPollTimeoutMax": 120
          }
        }
      }
    },
    "196": {
      "profId": 49757,
      "epId": 196,
      "devId": 512,
      "inClusterList": [1],
      "outClusterList": [1],
      "clusters": {
        "genPowerCfg": {
          "dir": {
            "value": 3
          },
          "attrs": {}
        }
      }
    }
  },
  "_id": "NTL0CCerOSkj7y3a"
}
```

### CODE REVIEW

**Comment by Koenkk** [comment]
Does this work with the same home assistant configuration as the keep home sv01?

**Comment by snjoetw** [comment]
Yep it does. I'll also make another PR to update homeassistant.js

File: devices.js [hunk 1]
```diff
@@ -2706,6 +2706,23 @@ const devices = [
         },
     },
 
+    // Gear Axis
+    {
+        zigbeeModel: ['Gear'],
+        model: 'Gear',
```
Comment (Koenkk) [inline devices.js]: Is there some model number written on the device? This is usually what we will put here.

Comment (snjoetw) [reply devices.js]: Just check it should be GR-ZB01-W, let me update it

**Comment by Koenkk** [comment]
Thanks again!

**Pull request merged**